packages/applicant/src/components/description-list/DescriptionList.tsx

```diff
@@ -2,7 +2,11 @@ import Link from 'next/link';
 import { useRouter } from 'next/router';
 import { FC } from 'react';
 
+const INDIVIDUAL = 'I am applying as an Individual';
```

Comment:
```suggestion
const INDIVIDUAL = 'I am applying as an individual';
```
Also probably a good idea to pull this into a constants file since it's used in multiple places. Should probably update the MQ flow with the constant as well